packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -169,6 +174,7 @@ contract Connext is
     tokenRegistry = TokenRegistry(_tokenRegistry);
     wrapper = IWrapped(_wrappedNative);
     EMPTY = hex"c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470";
+    maxRouters = 5;
```

Comment:
This value was picked arbitrarily from the discussions in the multipath specs. Is it ok to have this initialization or should we set this by a new parameter in `initialize`?